mutiny-core/Cargo.toml

```diff
@@ -0,0 +1,77 @@
+cargo-features = ["per-package-target"]
+
+[package]
+name = "mutiny-core"
+version = "0.2.4"
+edition = "2021"
+authors = ["Tony Giorgio <tony@mutinywallet.com>", "benthecarman <ben@mutinywallet.com>"]
+# forced-target = "wasm32-unknown-unknown"
+
+#[lib]
+#crate-type = ["cdylib"]
```

Comment:
think we can get rid of these

Reply:
Good catch. I left `forced-target` on for now since we still need wasm for core.